### DISCUSSION TITLE
Query Ring battery level in shortcut

### DISCUSSION THREAD

**Discussion by matt3274** [dgreif/ring#1024] [Q&A] [answered]
Is there any way within an iPhone shortcut (or home automation) to query the battery level of a ring doorbell?  I'm trying to have it notify me if a battery level gets below 20%, but can't seem to query the battery level property from the device.

**Comment by dgreif** [comment, marked as answer]
It's visible in the settings of the device in the Home app, but I don't think there is any way to set up notifications for low battery. That would be a good feature request to send Apples way 😄

**Reply by matt3274** [reply]
Thanks for the quick reply.  Glad to know I wasn't missing something obvious.  I was pulling my hair out trying to figure it out.  Thanks!
